website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -120,12 +130,94 @@ resource "vcd_nsxt_edgegateway" "nsxt-edge" {
 ```
 
 
+## Example Usage (Assigning NSX-T Edge Gateway to VDC group)
+
+```hcl
+data "vcd_nsxt_edge_cluster" "secondary" {
+  name = "edge-cluster-two"
+}
+
+data "vcd_external_network_v2" "nsxt-ext-net" {
+  name = "nsxt-edge"
+}
+
+data "vcd_vdc_group" "group1" {
+  name = "existing-group"
+}
+
+data "vcd_org_vdc" "vdc-1" {
+  name = "existing-group"
+}
+
+resource "vcd_nsxt_edgegateway" "nsxt-edge" {
+  org      = "my-org"
+  owner_id = data.vcd_vdc_group.group1.id
+
+  # VDC Group cannot be created directly in VDC Group - it 
+  # must originate in some VDC (belonging to destination VDC Group)
+  #
+  # `starting_vdc_id` field is optional. If only VDC Group 
+  # ID is specified in `owner_id` field - this resource 
+  # will will pick a random member
```

Comment:
Does it really pick a random member? If yes, please add this explanation to the documentation of the field itself below.

Reply:
Thank you for adding to the description of the field, looks great!